external/wakaama/examples/client/lwm2mclient.c

```diff
@@ -1216,66 +1218,66 @@ pthread_addr_t client_main(void)
 #endif
 	if (NULL == objArray[0]) {
 		fprintf(stderr, "Failed to create security object\r\n");
-		return -1;
+		return NULL;
 	}
 	data.securityObjP = objArray[0];
 
 	objArray[1] = get_server_object(serverId, "U", lifetime, false);
 	if (NULL == objArray[1]) {
 		fprintf(stderr, "Failed to create server object\r\n");
-		return -1;
+		return NULL;
 	}
 
 	objArray[2] = get_object_device();
 	if (NULL == objArray[2]) {
 		fprintf(stderr, "Failed to create Device object\r\n");
-		return -1;
+		return NULL;
 	}
 
 	objArray[3] = get_object_firmware();
 	if (NULL == objArray[3]) {
 		fprintf(stderr, "Failed to create Firmware object\r\n");
-		return -1;
+		return NULL;
 	}
 
 	objArray[4] = get_object_location();
 	if (NULL == objArray[4]) {
 		fprintf(stderr, "Failed to create location object\r\n");
-		return -1;
+		return NULL;
 	}
 
 	objArray[5] = get_test_object();
 	if (NULL == objArray[5]) {
 		fprintf(stderr, "Failed to create test object\r\n");
-		return -1;
+		return NULL;
 	}
 
 	objArray[6] = get_object_conn_m();
 	if (NULL == objArray[6]) {
 		fprintf(stderr, "Failed to create connectivity monitoring object\r\n");
-		return -1;
+		return NULL;
 	}
 
 	objArray[7] = get_object_conn_s();
 	if (NULL == objArray[7]) {
 		fprintf(stderr, "Failed to create connectivity statistics object\r\n");
-		return -1;
+		return NULL;
```

Comment:
Dear dr-venkman
Thanks for reviewing. We fully agreed your opinion and now we fix it.
"-1" returns in error case and "0" returns in normal case.